monkey/monkey_island/cc/resources/root.py

```diff
@@ -16,9 +15,6 @@
 
 
 class Root(flask_restful.Resource):
-    def __init__(self):
```

Comment:
This makes me nervous. Maybe we don't want the `flask_restful.Resource` constructor to run. I guess if blackbox tests pass, this is ok.

Reply:
Tests pass.